docs/antrea-network-policy.md

```diff
@@ -625,6 +626,11 @@ All of the above commands produce output similar to what is shown below:
     test-anp   securityops   5          5s
 ```
 
+### Audit logging for Antrea-native policies
```

Comment:
i think we can place this right after ClusterGroup section.. this seems like an odd place